test/screener/tests/dropdown.js

```diff
@@ -0,0 +1,14 @@
+const Steps = require('screener-runner/src/steps')
+const selector = require('../utils').selector
+
+module.exports = {
+  'DropdownExampleSingleSelection.shorthand': [
+    {
+      name: 'Shows List',
+      steps: new Steps()
+        .click(selector('trigger-button'))
```

Comment:
Let's use `className` from them matching slot (i.e. `ui-dropdown__trigger-button`), this will allow to avoid changes in examples